docs/src/navigation/subNavigationDeprecated/subNavigationDeprecated.js

```diff
@@ -1,15 +1,13 @@
-
 import MediaQuery from 'react-responsive';
```

Comment:
Are we planning on renaming the component to indicate it is deprecated?

Reply:
Hard deprecation. We should no longer be using this from here and out.

I don't think we should rename these in `./src`, but in `./docs/src` my thinking was/is to better identify what is deprecated. I could be wrong. Let me know.